temporal-testing-junit4/src/main/java/io/temporal/testing/TestWorkflowRule.java

```diff
@@ -139,6 +141,7 @@ public static Builder newBuilder() {
     private boolean useExternalService;
     private boolean doNotStart;
     private long testTimeoutSeconds;
+    private long initialTimeMillis = 0;
```

Comment:
We follow the pattern of not providing default values to fields as they are already initialized automatically in Java. So I would change it to:
```
private long initialTimeMillis;
```

Reply:
Thanks! My initial though was to have those zeros kind of align with the `currentTime = initialTime == 0 ? System.currentTimeMillis() : initialTime` logic from the `SelfAdvancingTimerImpl` but now I see it really looks inconsistent with the other code. Updated.